test.js

```diff
@@ -5,8 +5,10 @@ const fs = sandboxedFs.bind('./test');
 
 fs.readFile('file.ext', (err, data) => {
   if (err) return console.log('Cannot read file');
+  console.log(data.toString());
 });
 
 fs.readFile('../../file.ext', (err, data) => {
   if (err) return console.log('Cannot read file');
```

Comment:
Or you asked about whether it should be done in this PR or another? If that's what you meant, yeah, better to do it in a separate commit since it is not a lint-related issue.